openvdb/INSTALL

```diff
@@ -64,8 +69,10 @@ Other versions might work but have not been tested.
 Installation
 ------------
 
-This documentation is for building with GNU Make, it is recommended to use
-CMake to build OpenVDB. See the build documentation here -
+This documentation is for building with GNU Make which is now deprecated, it
+is strongly recommended to use CMake to build OpenVDB as this will be removed
```

Comment:
How about "It is strongly recommended to use CMake to build OpenVDB as the Makefiles will be removed in an upcoming release"?